shared/fs/popups/row-action-popup-container.js

```diff
@@ -33,6 +33,7 @@ const mapStateToProps = (state: TypedState, {routeProps}) => {
         )
 
   return {
+    mimeType: Constants.mimeTypeFromPathItem(pathItem),
```

Comment:
I like naming intermediate props that don't get passed out from `mergeProps` with a `_` prefix. But no big deal.